src/components/dify-chatbot.module.css

```
@@ -0,0 +1,125 @@
+}
+
+:global(#dify-chatbot-container) {
+  position: fixed;
+}
+
+:global(#dify-chatbot-container[data-dify-load-state='error']) {
+  width: min(20rem, calc(100vw - 2rem));
+  max-width: 20rem;
+}
+
+:global(#dify-chatbot-container.dify-chatbot-load-failed::before) {
```

Stylelint (17.3.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)